config/locales/js-en.yml

```diff
@@ -82,6 +83,7 @@ en:
     label_collapse: "Collapse"
     label_collapsed: "collapsed"
     label_collapse_all: "Collapse all"
+    label_committed_at: "committed revision %{revision}"
```

Comment:
What you are describing sounds like a hell of a localization problem. Because we won't be able to just do:

````html
<a href="">{{ t('label_created') }}</a> {{ t('label_at') }} {{ getTheDate() }}
````

This is not translatable very well... the best thing would be something along the lines of `committed revision %{revision} at {date}`, because this offers the best translatability. But we lose our fine grained linkability, especially since we do not want HTML in our translations.